trade-finance-manager-api/src/v1/controllers/acbs.controller.js

```diff
@@ -184,7 +184,7 @@ const amendAcbsFacility = async (amendments, facility, deal) => {
 
   api.amendACBSfacility(payload, facility, deal)
     .then((acbsTaskLinks) => {
-      if (acbsTaskLinks.id) {
+      if (acbsTaskLinks && acbsTaskLinks.id) {
```

Comment:
is the second validation needed? i.e. `acbsTaskLinks` ? 
I thought `acbsTaskLinks.id` would check both `acbsTaskLinks` and `acbsTaskLinks.id`

Reply:
This is in case, `id` does not exists so the first condition would stop it there and prevent from crashing.

Reply:
it wouldn't crash :) - it would simply return `undefined`
in this scenario it would crash `acbsTaskLinks.anotherValue.id` - if `anotherValue` is undefined, then it would crash